service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java

```diff
@@ -1327,18 +1313,17 @@ public void updateOrderAdminPageInfoAndSaveReason(Long orderId, UpdateOrderPageA
     @Transactional
     public void updateOrderAdminPageInfo(UpdateOrderPageAdminDto updateOrderPageDto, Order order, String lang,
         String email) {
-        isOrderStatusCanceledOrDone(order);
-
         checkAvailableOrderForEmployee(order, email);
-
-        if (order.getOrderStatus() == OrderStatus.BROUGHT_IT_HIMSELF
-            && nonNull(updateOrderPageDto.getGeneralOrderInfo())) {
-            updateOrderDetailStatus(order, updateOrderPageDto.getGeneralOrderInfo(), email);
-        } else {
-            try {
-                updateOrderPageFields(updateOrderPageDto, order, email);
-            } catch (Exception e) {
-                throw new BadRequestException(e.getMessage());
+        if (!processRefundForOrder(order, updateOrderPageDto, email)) {
```

Comment:
I see that `processRefundForOrder` can throw an exception 
Should we add a catch block for this case?